scriptPath,detail,startTime,endTime,duration,result
/Users/tylan/Desktop/myAuty/Auty/scripts/scripts/quanzi/update_user_avatar_test.py,No exception in execute_script method.,2016-10-11 21:51:02.845375,2016-10-11 21:51:03.633071,0.787696,"{""result"":100,""avatarUrl"":""http://img2.imgtn.bdimg.com/it/u=1160161467,1922703035&fm=21&gp=0.jpg"",""nickName"":""runcheck**"",""resultDesc"":""成功""}
10919
passed
passed
passed
"
,Exception in execute_script method: [Errno 2] No such file or directory: '',2016-10-11 21:51:03.633223,2016-10-11 21:51:03.633550,0.000327,
